go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/AzureAD/microsoft-authentication-library-for-go
+module github.com/wayneforrest/microsoft-authentication-library-for-go
```

Comment:
Was this change intended?

Reply:
ooh .. I see what happened here 😂 I was forking the repo. I will fix this.